internal/pkg/manifest/rd_web_svc.go

```diff
@@ -60,15 +61,48 @@ func (c *RequestDrivenWebServiceNetworkConfig) IsEmpty() bool {
 	return c.VPC.isEmpty()
 }
 
-// RequestDrivenWebServicePlacement represents where to place tasks for a Request-Driven Web Service.
-type RequestDrivenWebServicePlacement Placement
+// RequestDrivenWebServicePlacement represents what types of subnets to place tasks.
+type RequestDrivenWebServicePlacementString PlacementString
```

Comment:
This is a bit of tangent, but an opportunity to simplify:
I recently learned from Amit on the App Runner team that `placement: public` is actually supported by App Runner and doesn't require a nat gateway in that scenario.

So my point is we can actually completely get rid of `RequestDrivenWebServicePlacementString` and use `PlacementArgOrString` 😍